### PR TITLE
secure xml

### PR DESCRIPTION
**Pull request by viqueen**
I found out that this library is open to some xml vulnerabilities (xxe / ssrf)
This pull request ensures both the MethodBase and PostRequestPars types are relying on xml secure processing

### CODE REVIEW

**Review by douglm — CHANGES_REQUESTED** [review]
Great change - thank you. Could you drop the change to gitignore and reindent to 2 spaces?

I'm about to roll out a new release and I'd like this in there.

File: mvnvm.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+mvn_version=3.5.4
```
Comment (viqueen) [inline mvnvm.properties:1]: we use the following to manage maven versions across developers http://mvnvm.org/
thought it might be useful here

Comment (douglm) [reply mvnvm.properties:1]: Thanks - I'll take a look

**Review by douglm — APPROVED** [review]
This all looks good - thanks

**Pull request merged**